view/theme/vier/templates/comment_item.tpl

```diff
@@ -13,32 +13,32 @@
 				<input type="hidden" name="preview" id="comment-preview-inp-{{$id}}" value="0" />
 				<input type="hidden" name="post_id_random" value="{{$rand_num}}" />
 
-				<div class="comment-edit-photo" id="comment-edit-photo-{{$id}}" >
+				<div class="comment-edit-photo" id="comment-edit-photo-{{$id}}">
 					<a class="comment-edit-photo-link" href="{{$mylink}}" title="{{$mytitle}}"><img class="my-comment-photo" src="{{$myphoto}}" alt="{{$mytitle}}" title="{{$mytitle}}" /></a>
 				</div>
 				<div class="comment-edit-photo-end"></div>
-				<textarea id="comment-edit-text-{{$id}}" class="comment-edit-text-empty" name="body" onFocus="commentOpen(this,{{$id}});">{{$comment}}</textarea>
+				<textarea id="comment-edit-text-{{$id}}" class="comment-edit-text-empty" name="body" placeholder="{{$comment}}" onFocus="commentOpen(this,{{$id}});"></textarea>
 				{{if $qcomment}}
-					<select id="qcomment-select-{{$id}}" name="qcomment-{{$id}}" class="qcomment" onchange="qCommentInsert(this,{{$id}});" >
+					<select id="qcomment-select-{{$id}}" name="qcomment-{{$id}}" class="qcomment" onchange="qCommentInsert(this,{{$id}});">
 					<option value=""></option>
 				{{foreach $qcomment as $qc}}
-					<option value="{{$qc}}">{{$qc}}</option>				
```

Comment:
The removal of white spaces can be automatically done. Plus maybe it is time to use PHP-CS ?